packages/core/src/components/graphs/area-stacked.ts

```diff
@@ -64,7 +60,7 @@ export class StackedArea extends Component {
 		// D3 area generator function
 		this.areaGenerator = area()
 			// @ts-ignore
-			.x((d) => mainXScale(new Date(d.data.sharedStackKey)))
+			.x((d) => mainXScale(isTimeSeries ? new Date(d.data.sharedStackKey) : d.data.sharedStackKey))
```

Comment:
please test with just `d.data.sharedStackKey` and see whether this still works.

With the sanitize PR I just merged that should be good enough